setup.sh

```diff
@@ -183,24 +183,40 @@ else
 fi
 
 ########## Always install MAVSDK ##########
-echo "Downloading the latest release of mavsdk"
-release_info=$(curl -s https://api.github.com/repos/mavlink/MAVSDK/releases/latest)
-# Assumes arm64
-download_url=$(echo "$release_info" | grep "browser_download_url.*arm64.deb" | awk -F '"' '{print $4}')
-file_name=$(echo "$release_info" | grep "name.*arm64.deb" | awk -F '"' '{print $4}')
-
-if [ -z "$download_url" ]; then
-    echo "Download URL not found for arm64.deb package"
-    exit 1
-fi
+# Check if we are on 20.04 or 22.04
+if [ "$(lsb_release -cs)" = "focal" ]; then
+	echo "Ubuntu 20.04 detected, building MAVSDK from source"
+	pushd .
+	sudo rm -rf ~/code/MAVSDK
+	git clone https://github.com/mavlink/MAVSDK.git ~/code/MAVSDK
+	cd ~/code/MAVSDK
+	git submodule update --init --recursive
```

Comment:
```suggestion
	git clone --recurse-submodules --depth=1 --shallow-submodules https://github.com/mavlink/MAVSDK.git ~/code/MAVSDK
	cd ~/code/MAVSDK
```

Reply:
the depth=1 and shallow-submodules options speeds up the download a lot, so it doesn't have to pull and index all of the repo history